wandb/sdk/wandb_config.py

```diff
@@ -233,12 +232,15 @@ def _sanitize_dict(
         return sanitized
 
     def _sanitize(self, key, val, allow_val_change=None):
+        # refuse all WBValues which is all Media and Histograms
```

Comment:
nit: can we leave a TODO here to enable this in the future?

Reply:
Absolutely, I actually have a branch with those changes started